### DISCUSSION TITLE
Welcome to quick_trade Discussions!

### DISCUSSION THREAD

**Discussion by VladKochetov007** [VladKochetov007/quick_trade#246] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Michelious** [comment]
Hello,

I am trying to use the code. When I execute realtime_trading, the stragey returns signals of buy or sell, but before going to create the order, the code inspects the last element of the array _converted, which always is nan.  So, it never creates the order.

What is the purpose of the utils.convert() function?

Regards

**Reply by VladKochetov007** [reply]
Hi, utils.convert returns not only-nan list, this function is really simple and it needs to inform us when signal was changed

```python
def convert(data: PREDICT_TYPE_LIST) -> CONVERTED_TYPE_LIST:
    ret: List[Any] = list(data.copy())
    e: int
    for e, i in enumerate(data[1:]):
        if i == data[e]:
            ret[e + 1] = nan
    return ret
```
When the signal has changed, we understand that we need to enter the trade

**Comment by Michelious** [comment]
Hi Vlad,

How is it possible to combine two strategies at the same time?

**Reply by VladKochetov007** [reply]
https://vladkochetov007.github.io/quick_trade/#/docs/quick_trade/trading_sys?id=multi_strategy_collider

strategy_collider method or multi_strategy_collider

**Comment by Manojbhat09** [comment]
Hi Vlad, 

Have you found success in using this tool whats the margin did you gain in % weekly/monthly?


Regards,

**Reply by VladKochetov007** [reply]
Hello! Actually, no. 
I'm developing new and more powerful system.
I tried to trade, but it needs more validation and testing steps
